src/main/java/nextstep/courses/domain/session/SessionStatus.java

```diff
@@ -0,0 +1,11 @@
+package nextstep.courses.domain.session;
+
+public enum SessionStatus {
+  READY,
+  IN_PROGRESS,
+  END;
+
+  public boolean isInProgressStatus() {
+    return this.equals(IN_PROGRESS);
```

Comment:
```suggestion
    return this == IN_PROGRESS;
```
enum 타입이므로 동일성이 보장됩니다.